charmhub_lp_tools/reports.py

```diff
@@ -0,0 +1,207 @@
+import collections
+import json
+import logging
+import operator
+import os
+
+from datetime import datetime
+from typing import List
+try:
+    from zoneinfo import ZoneInfo
+except ImportError:
+    from backports.zoneinfo import ZoneInfo
+
+import humanize
+
+from jinja2 import Environment, FileSystemLoader, select_autoescape
+from prettytable import PrettyTable
+
+from .launchpadtools import TypeLPObject
+
+
+__THIS__ = os.path.dirname(os.path.abspath(__file__))
```

Comment:
I'm slightly dubious about the name of this var?  Does it need to be double-underscored?  Could it be something other that "THIS"; i.e. DIR__file__ or something descriptiive?  Or if it's never going to change, just eliminate the __THIS__ and move it into the `FileSystemLoader` call?

Reply:
ack, +1